src/Swashbuckle.AspNetCore.SwaggerGen/SchemaGenerator/OpenApiSchemaExtensions.cs

```diff
@@ -230,6 +238,16 @@ private static void ApplyStringLengthAttribute(OpenApiSchema schema, StringLengt
             schema.MaxLength = stringLengthAttribute.MaximumLength;
         }
 
+        private static void ApplyReadOnlyAttribute(OpenApiSchema schema, ReadOnlyAttribute readOnlyAttribute)
+        {
+            schema.ReadOnly = readOnlyAttribute.IsReadOnly;
+        }
+
+        private static void ApplyDescriptionAttribute(OpenApiSchema schema, DescriptionAttribute descriptionAttribute)
+        {
+            schema.Description = descriptionAttribute.Description;
```

Comment:
I wonder if we should change this to `??=` so we don't overwrite any existing value?